### PR TITLE
[16] Render the related product list on the HTML page with JavaScript

### PR DESCRIPTION
**Pull request by Samju200**
## Task: JavaScript task`
- `use array of object to render the related product list on the Html
javascript-3

`

### CODE REVIEW

**Comment by ToluAkin** [comment]
@Samju200 Do you still need help figuring out the updated task?

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Samju200 Check the instruction again [here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/320)

File: submissions/adeyemiJuwon/javascript/productlist.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    {
+      name: "11 Inch Laptop Bag Tablet Zipper Pouch Sleeve for MacBook Air",
+      imgUrl: "../introductionToHtml/img/bag 1.jpg",
+      price: "£5.95"
+    }
+  ];
+
+  const newList = productList.forEach((product) => {
+    console.log(product)
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/javascript/productlist.js]: Log this in the right format - as stated in the task

File: submissions/adeyemiJuwon/javascript/productlist.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    productPrice: "£4.26 GBP"
+  },
+  {
+    productName: "11 Inch Laptop Bag Tablet Zipper Pouch Sleeve for MacBook Air",
+    imageUrl: "img/bag1.jpg",
+    productPrice: "£5.95"
+  }
+];
+
+const newList = productList.forEach((product) => {
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/javascript/productlist.js:21]: There is no need for variable declaration `const newList` cos it is not been used. Also, format your code(line 22)

File: submissions/adeyemiJuwon/javascript/productlist.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Product list</title>
+</head>
+<body>
+  <div class="related-product">
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/javascript/productlist.html:9]: Also, line 9- 12 isn't necessary

**Review by adaugochi — APPROVED** [review]

**Pull request merged**